_includes/manuals/1.12/5.8.3_secret_token.md

```diff
@@ -0,0 +1,8 @@
+
+The last file required to make a Foreman cluster work is __$app_root/tmp/secret_token.rb__. This file should contain a secret token that is modeled after __$app_root/config/initializers/local\_secret\_token.rb__. __$app_root/tmp/secret_token.rb__ is used by __$app_root/config/initializers/secret_token.rb__ to sign cookies. Once you have created the new secret_token.rb, restart Foreman and clear Foreman's cache:
```

Comment:
You're right. I tried point 1 and it's showing the token in my local_secret_token, not the one in secret_token. I'll amend the entry shortly.